Source/Bifrost.Web/packages.config

```diff
@@ -3,17 +3,17 @@
   <package id="Antlr" version="3.5.0.2" targetFramework="net45" />
   <package id="jQuery" version="2.2.3" targetFramework="net451" />
   <package id="knockoutjs" version="3.4.0" targetFramework="net451" />
-  <package id="LessMsbuildTasks" version="1.0.7" targetFramework="net45" />
-  <package id="Microsoft.AspNet.SignalR.Core" version="2.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.SignalR.JS" version="2.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.SignalR.SystemWeb" version="2.2.0" targetFramework="net45" />
+  <package id="LessMsbuildTasks" version="1.0.10" targetFramework="net451" />

```

Comment:
Update Bifrost.Web.nuspec with all the new packages versions from this file. (Or remove the lines from the nuspec file and verify that the dependencies in the NuGet package stays the same, it should be picked up automatically..)